tests/core/dom/document.js

```diff
@@ -142,6 +134,20 @@ bender.test( appendDomObjectTests(
 			assert.areSame( 1, div.find( 'video' ).count(), 'Second edge element' );
 
 			assert.areSame( $frag, CKEDITOR.document._getHtml5ShivFrag(), 'Document fragment is cached' );
+		},
+
+		// (#910)
+		'test getScrollingElement': function() {
+			var scrollingElement = CKEDITOR.document.getScrollingElement(),
+				nativeEl = document.scrollingElement || document.documentElement || document.body;
+
+			if ( !( CKEDITOR.env.ie && CKEDITOR.env.version < 9 ) ) {
+				// IE8 doesn't support isEqualNode
+				assert.isTrue( scrollingElement.$.isEqualNode( nativeEl ) );
```

Comment:
It should be rather `===` than `isEqualNode`. We are not looking for equal nodes, but checking if nodes are **the same**. [Example on MDN](https://developer.mozilla.org/en-US/docs/Web/API/Node/isEqualNode#Example) clearly shows this difference. Introducing this change would also allow us to test it in old IEs.